extension/source/App.ts

```diff
@@ -206,14 +204,9 @@ export default class App {
     this.incrementWalletAddressLoading();
 
     try {
-      const creationTx = await BlsWallet.signCreation(
-        this.state.privateKey,
-        env.NETWORK_CONFIG.addresses.verificationGateway,
-        this.provider,
-      );
-
-      const createResult = await this.aggregator.createWallet(creationTx);
+      throw new Error('App: Wallet creation not implemented');
```

Comment:
We're not doing an explicit create anymore, so the goal is to just remove this `createWallet` method entirely.

Reply:
I still think there are cases where someone may want to non-lazily create a wallet, but I imagine those are not needed anytime soon.

I will add tasks to this PR to remove this, similar functionality in aggregator, and create an issue for manual wallet creation we can evaluate later.

Reply:
Removed wallet creation paths and logic from agg and ext.

Reply:
> I still think there are cases where someone may want to non-lazily create a wallet, but I imagine those are not needed anytime soon.

True. Should be ok with just submitting an empty operation from the client for that though. Keeps the interface simple.